From 2f8b6fe5e51aa592637a0d9c3ad817978d8a2157 Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Thu, 13 Oct 2016 15:25:45 -0700
Subject: [PATCH] joyent/node-fast#2 update dependencies to support Node v4
 Reviewed by: Brian Bennett <brian.bennett@joyent.com> Approved by: Alex
 Wilson <alex.wilson@joyent.com>

---
 CHANGES.md   | 4 ++++
 package.json | 4 ++--
 2 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/CHANGES.md b/CHANGES.md
index 108d54a..8a4ba05 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -4,6 +4,10 @@
 
 No changes.
 
+## v2.2.0
+
+* #2 update dependencies to support Node v4
+
 ## v2.1.0
 
 * #1 need limited support for servers that send null values
diff --git a/package.json b/package.json
index ca21e25..b675ef8 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
 	"name": "fast",
 	"description": "streaming JSON RPC over TCP",
-	"version": "2.1.0",
+	"version": "2.2.0",
 	"main": "./lib/fast.js",
 	"repository": {
 		"type": "git",
@@ -14,7 +14,7 @@
 		"crc": "0.3.0",
 		"extsprintf": "^1.3.0",
 		"jsprim": "^1.2.2",
-		"kang": "^1.0.1",
+		"kang": "^1.1.0",
 		"lstream": "0.0.4",
 		"microtime": "2.0.0",
 		"posix-getopt": "1.2.0",
-- 
2.21.0

